tasks/main.yml

```diff
@@ -57,8 +57,11 @@
 - name: Ensure Sonar is running and set to start on boot.
   service: name=sonar state=started enabled=yes
 
-- name: Allow Sonar time to build on first start.
-  pause: seconds=180
+- name: Ensure Sonar is started.
+  wait_for:
+    path: /usr/local/sonar/logs/sonar.log
```

Comment:
As this role unzip sonar into `/usr/local/` I think that's ok for all.

Source: https://github.com/geerlingguy/ansible-role-sonar/blob/master/tasks/main.yml#L24